addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -368,6 +372,59 @@ const Wysiwyg = Widget.extend({
     redo: function () {
         this.odooEditor.historyRedo();
     },
+    /**
+     * Start or resume the date field changes muation observers.
+     *
+     * Necessary to keep all copies of a given field at the same value throughout the page.
+     */
+    _observeDateFieldChanges: function () {
```

Comment:
As mentionned in a video call we did, I would make the name of the function generic for covering all the cases where we need to duplicate the data of a node (i.e. all odoo fields and translate).

Reply:
I can do that